libs/components/forms/src/lib/modules/input-box/input-box.component.html

```diff
@@ -76,92 +76,90 @@
 
```

Comment:
the below chunk of code adding `sky-input-box-errors` on line 182/179 is the only relevant change in this file.